### ISSUE TITLE
Series on svtplay with a single item doesn't download anything when using -A

### ISSUE THREAD

**Issue by Jacalz** [spaam/svtplay-dl#956]
If you are using -A when downloading a series with only one item it doesn't download anything at all. This is very annoying when you tell it to download three different series where some of them only have one item and not more.

**Comment by Sopor-** [comment]
@Jacalz Could you show us an example how you do it?
So you use -A with several URLs? Does svtplay-dl really support that? I know it work on single episodes but with -A i'm not sure or you have manage to get it to work?

**Comment by myowncloudservice** [comment]
I have try to use more the one URLs in svtplay-dl with ; between each URLs, but svtplay-dl will crash.

In Youtube-dl you can do this with a batch file

**Comment by Sopor-** [comment]
Have you tried space instead of `;`?

**Comment by myowncloudservice** [comment]
, don't work either, only the first url will be downloaded, then the svtplay-dl application shutdown.

**Comment by Jacalz** [comment]
Sorry for being late on answering, I missed this in the inbox :grimacing:
Anyhow, yes it does support more links, and this is an example of what I would do: 
(The first and third links are series and number two and four are just one video)

`
svtplay-dl -A -S https://www.svtplay.se/det-stora-racet https://www.svtplay.se/video/2557384/ata-sova-do https://www.svtplay.se/video/19052735/duellen/duellen-sasong-2-15-sep-02-00 https://www.svtplay.se/video/14362324/potatishandlaren
`

Downloading this, downloads the series with multiple episodes but the movies with only one video don't get downloaded. Downloading the two movies without -A works: 

`
svtplay-dl -S https://www.svtplay.se/video/2557384/ata-sova-do https://www.svtplay.se/video/14362324/potatishandlaren
`

**Comment by Sopor-** [comment]
`-A` is used to download all episodes of a program. When downloading single episodes you need to leave out the `-A`. You can easy do that by using `svtplay-dl -A -S URL & svtplay-dl -S URL`

**Comment by Jacalz** [comment]
@spaam Could you please fix this? I have my config set to always download all episodes and it is very annoying to have it not download single series. In my opinion it should be possible to download single files while telling it to download all. All episodes are still one and not zero when there is one episode so logical it seems good to not just exit without any errors...

**Comment by Sopor** [comment]
Remove `-A` from the config and add it when you want to download all. Using -A when downloading one episode make no sense.

**Comment by Sopor** [comment]
This is one of the reasons i want to disable config settings with a switch `--noconfig` so it is possible to use svtplay-dl with other ones. Maybe even disable single switches `-dA` will ignore -A from the config file.

**Comment by Jacalz** [comment]
@Sopor I agree with you about being able to disable config and maybe disabling individual switches. But I see absolutely NO reason that single episodes shouldn’t be downloading when you tell it to download all. This is primarily a bug if you ask me and it would be great to get it fixed...

**Comment by Sopor** [comment]
> Downloading this, downloads the series with multiple episodes but the movies with only one video don't get downloaded. Downloading the two movies without -A works

Sorry, my bad. It is a bug :)
